build.gradle

```
@@ -0,0 +1,41 @@
+plugins {
+    id 'java'
+    id 'application'
+    id 'com.github.johnrengelman.shadow' version '7.1.2'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.10.0'
+    testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.10.0'
+}
+
+test {
+    useJUnitPlatform()
+
+    testLogging {
+        events "passed", "skipped", "failed"
+
+        showExceptions true
+        exceptionFormat "full"
+        showCauses true
+        showStackTraces true
+        showStandardStreams = false
+    }
+}
+
+application {
+    mainClass.set("duke.Geng")
+}
+
+shadowJar {
+    archiveBaseName = "geng"
+    archiveClassifier = null
+}
+
+run{
+    standardInput = System.in
+}
```

```suggestion
run {
    standardInput = System.in
}
```
I like how you followed the coding standard by using proper indentation and spacing for the run block. However, I noticed that this specific line is missing one space between run and {. Adding the space would make it consistent with the rest of the codebase, which adheres well to the standard. Great job overall!